QuotesApp/Core/MainScreen/MainScreenView.swift

```diff
@@ -49,18 +48,10 @@ struct MainScreenView: View {
         }
     }
     
-    private func buildLoadedListView(books: [BookItem]) -> some View {
+    private func buildLoadedListView(books: [Domain.BookItem]) -> some View {
         VStack {
-            ScrollView(showsIndicators: false) {
-                LazyVStack(alignment: .leading) {
-                    ForEach(books) { book in
-                        BookListRowView(book: book)
-                            .padding(.horizontal)
-                    }
-                }
-            }
-            .safeAreaInset(edge: .top) {
-                Color.clear.frame(height: 30)
+            BookListView(books: books, showQuotesNumber: true) {_ in 
```

Comment:
moglibysmy w przyszlosci dodac swiftlint zeby wylapywal nam takie drobne rzeczy 
```suggestion
            BookListView(books: books, showQuotesNumber: true) { _ in 
```